pyproject.toml

```
@@ -43,7 +43,7 @@ uvicorn = [
     {version = "^0.24.0", python = ">=3.12"},
     {version = "^0.20.0", python = "<3.12"},
 ]
-watchdog = "^2.3.1"
+watchdog = "^4.0.0"
```

```suggestion
watchdog = ">=2.3.1,<4.1.0"
```

since we didn't break compatibility with 2.3.1, we shouldn't restrict users from the older versions